components/DropdownHeader.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { FunctionComponent, useRef } from "react";
+import HeaderItems from './HeaderItems';
+import Image from "next/image";
+import Icon from "@mdi/react";
+import { mdiClose } from "@mdi/js";
+
+const menuItems = [
+  {
+    title: 'Quem Somos',
+    url: '#',
+    submenu: [
+      {
+        title: 'Nosso time',
+        url: '#',
+      },
+    ],
+  },
+  {
+    title: 'Soluções',
+    url: '#services',
+    submenu: [
+      {
+        title: 'Aplicações personalizadas',
+        url: '#',
+      },
+      {
+        title: 'Consultoria em desenvolvimento',
+        url: '#',
+      },
+      {
+        title: 'Soluções Sharpy',
+        url: '#',
+      },
+    ],
+  },
+  {
+    title: 'Carreiras',
+    url: '#',
+  },
+  {
+    title: 'Fale Consco',
+    url: '#',
+  },
+]
+
+interface Props {
+  mobile?: Boolean;
+  onChildClick?: () => void;
+}
+
+const DropdownHeader: FunctionComponent<Props> = (props) => {

```

Comment:
I think that DropdownHeader is not the best name for this component, since renders a list of the nav items.